test/oracle.test.ts

```diff
@@ -0,0 +1,37 @@
+import { JsonRpcProvider } from "@ethersproject/providers";
+import { BigNumber } from "bignumber.js";
+
+import { Yearn } from "../src";
+import { ChainId } from "../src/chain";
+import { Integer, Usdc } from "../src/types";
+
+describe("oracle", () => {
+  let yearn: Yearn<ChainId>;
+  let tokenAddress: string;
+  beforeAll(async () => {
+    const chainId = 1;
+    const rpcUrl = "";
```

Comment:
does using `process.env.WEB3_PROVIDER` work here?

Reply:
Definitely.  I'll add it.